app/src/main/java/org/mozilla/tv/firefox/IntentValidator.kt

```diff
@@ -101,9 +98,9 @@ object IntentValidator {
         }
     }
 
-    private fun changeFetchDelaySecondsForQA(intent: SafeIntent) {
+    private fun changeFetchDelaySecondsForQA(intent: SafeIntent, context: Context) {
```

Comment:
super-nit: when `context` is a parameter, I like to put it first. Contexts are dangerous (memory leaks, access to everything, etc.) so it's important to highlight it when possible.

That being said, I think you should take the `PocketVideoFetchScheduler` into this method, rather than a Context: it's better to be explicit and scope access down as much as possible.